### PR TITLE
Migrate from Travis CI to Github Actions

### PR DESCRIPTION
**Pull request by victor-carvalho**

### CODE REVIEW

File: .github/workflows/quality.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+name: Quality
+
+on:
```
Comment (gabrielcorado) [inline .github/workflows/quality.yaml:3]: I guess we also want to run it on pushes to master, right?

File: .github/workflows/quality.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
+name: Quality
```
Comment (gabrielcorado) [inline .github/workflows/quality.yaml:1]: Curiosity: Why is it named `Quality`? (not `Test` or smth)

**Review by gabrielcorado — COMMENTED** [review]
Friendly reminder: squash the commits when merging 👍 .

File: .github/workflows/quality.yaml (hunk 2 above)
Comment (victor-carvalho) [reply .github/workflows/quality.yaml:1]: Just copied it from somewhere. But I can rename it to `Tests`

File: .github/workflows/quality.yaml (hunk 1 above)
Comment (victor-carvalho) [reply .github/workflows/quality.yaml:3]: Ok, I will enable it for pushs on master too

**Review by gabrielcorado — APPROVED** [review]

**Comment by victor-carvalho** [comment]
> Can we add the coverage report? The Makefile already supports coverage.

It should work as I'm reporting to coveralls the same way travis was doing.

**Review by rodopoulos — APPROVED** [review]

**Comment by rodopoulos** [comment]
LGTM

**Pull request merged**